docker-compose.yml

```diff
@@ -1,4 +1,3 @@
-version: '3'
```

Comment:
the version field has been obsolete for years. I don't want to see warnings anymore.